### ISSUE TITLE
Comparing ember objects

### ISSUE THREAD

**Issue by CoreyFedde** [ga-wdi-boston/capstone-project#855]
TL/DR Comparing objects worked. Then changed objects to ember objects. Now they aren't showing up as equivalent to each other.

I am encountering a problem with comparing ember objects to each other.

Previously, I've been using a comparison helper to compare an object I create in a component (selectedLoan) to objects I am getting from the backend (loan). 
Code:
```handlebars
{{#each loans as |loan|}}
              {{#if (eq selectedLoan loan)}}
                <li>{{loan-list/card loan=loan}}</li>
              {{/if}}
            {{/each}}
```
This way, I had the template loop through each loan and build a card to display it when it matches the selectedLoan I created on the front end. This way worked while I was using POJOs.

However, I've needed to convert the object I was building in my component to an ember object. And is now not working. The strange part is when I print out both the loan and the selectedLoan, they are exactly the same, but they are not equating in the helper.

Is there a way to compare the two to determine which loans to display based on a selection?

**Comment by scottyscripts** [comment]
Have you tried comparing the id's of both of these objects? `{{#if (eq selectedLoan.id loan.id)}}` ?
The two objects will not evaluate to `===` because they are different ember objects, but we can compare their id attributes

**Comment by CoreyFedde** [comment]
I can't access an id on selectedLoan

**Comment by payne-chris-r** [comment]
Where are we at with this?

**Comment by CoreyFedde** [comment]
One specific solution to my problem was something Scott showed me.

In my component js file, I could use store to peek the record to find the full ember object based on the Id. The code looks like this:
```js
export default Ember.Component.extend({
  store: Ember.inject.service(),
  actions: {
    selectLoan(loanId) {
        let loan = this.get('store').peekRecord('loan', loanId)
      return this.sendAction('selectLoan', loan)
    },
  },
});
```
But, it looks like if this is necessary, it may be the result of one component trying to do too much. The best solution is probably to break it up and carry some of what I was trying to do into another view.
